modules/SofaGraphComponent/SceneCheckAPIChanges.cpp

```diff
@@ -112,7 +113,7 @@ void SceneCheckAPIChange::doCheckOn(Node* node)
 void SceneCheckAPIChange::installDefaultChangeSets()
 {
     addHookInChangeSet("17.06", [](Base* o){
-        if(o->getClassName() == "RestShapeSpringsForceField" && o->findData("external_rest_shape")->isSet())
+        if(o->getClassName() == "RestShapeSpringsForceField" && o->findLink("external_rest_shape")->getSize() != 0)
```

Comment:
This is messing with my brain again ><
What happens if there is no `external_rest_shape` parameter in my RestShapeSpringsForceField?

Reply:
A suspect a lot of bad things happens if you start removing the default Data field and links at runtime. 
In this case it crash.

Reply:
@guparan If you have the time to add the test do it...I'm fully busy until my presentation at Toulouse thursday.

Reply:
Well, finally I started qtcreator and I was on the right branch...so made the fix.